index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>webport</title>
+    <title>My Portfolio - HomePage</title>
```

Comment:
- [ ] [OPTIONAL] You have done a great job implementing the accessibility for your site :fire: but I would suggest being a bit more descriptive with your portfolio title because "My Portfolio" can be too general, maybe adding your name and/or what you do could be a good idea (e.g., @DelhinRharl's Portfolio or @DelhinRharl Web Developer) :pushpin: :wink:

Reply:
alright...thanks so much

Reply:
alright...thanks so much